### DISCUSSION TITLE
Tracking certain node or group of nodes

### DISCUSSION THREAD

**Discussion by TheMalaland** [talmolab/sleap#803] [Help!] [answered]
Hello i'm  currently working on dog movement prediction, focusing efforts on predicting movement on dog's tail,

Since i'm kinda new using sleap, i would to know if there's a way of

-Using a full body single instance dog skeleton
-predict the tail movement and get the results on a txt or output.log

**Comment by roomrys** [comment, marked as answer]
Hi @TheMalaland,

Happy to help you get started with some SLEAP! 

A full body single instance dog skeleton is definitely doable. We have a few pointers on skeleton design [here](https://sleap.ai/guides/skeletons.html?highlight=skeleton). If you want to get into specifics when designing your skeleton, feel free to ask - we are happy to help. The entire skeleton will be tracked, but you can focus on just a group of nodes using SLEAP's Analysis HDF5 file (when you arrive at the analysis step of your project).

For analysis, SLEAP currently outputs an HDF5 format (File > Export Analysis HDF5...). To help users navigate the HDF5, we have an [Analysis Examples](https://sleap.ai/notebooks/Analysis_examples.html) notebook that can be opened in Colab. From there, you should be able to extract any points you need (and also convert to whichever format you prefer). 

Let us know if this helps or if you have any more questions.

Thanks,
Liezl
